src/app/components/shared/icon-link/IconLink.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import Icon from 'nav-frontend-ikoner-assets';
+
+import './iconLink.less';
+
+export const IconLink = ({ iconKind, to }) => (
+	<div className="iconLink">
+		<Icon kind={iconKind} size="15" />
+		<Link className="iconLink__link" to={to}>
+			Les mer om terminbekreftelsen
```

Comment:
For å gjøre komponenten mer generell kunne kanskje denne teksten kommet inn som en prop?

Reply:
Jepp, det burde det absolutt være. Fikset.